xarray/tests/test_plot.py

```diff
@@ -2421,6 +2422,8 @@ def test_get_axis(figsize, aspect, size):
 
 @requires_cartopy
 def test_get_axis_cartopy():
+    import cartopy as ctpy
+
```

Comment:
you could also add this to the top of the module:
```python
try:
    import cartopy as ctpy
except ImportError:
    ctpy = None
```

Reply:
done